src/cleanlab_tlm/__about__.py

```diff
@@ -1,2 +1,2 @@
 # SPDX-License-Identifier: MIT
-__version__ = "1.1.1"
+__version__ = "1.1.2"
```

Comment:
```suggestion
__version__ = "1.1.3"
```

Reply:
v1.1.2 is not yet released. I've merged main into this branch, so this will still be a part of 1.1.2 when released.